cmd/admin-revoker/main.go

```diff
@@ -164,13 +164,19 @@ func main() {
 
 				tx, err := dbMap.Begin()
 				if err != nil {
-					tx.Rollback()
+					rollbackErr := tx.Rollback()
+					if rollbackErr != nil {
+						fmt.Printf("In addition, the transaction rollback failed: %v\n", rollbackErr)
```

Comment:
It looks like this doesn't come "in addition" to any previously printed message. How about:

```
if err != nil {
  rollbackErr := tx.Rollback()
  cmd.FailOnError(fmt.Sprintf("Couldn't begin transaction: %s, (rollback result: %v)", err, rollbackErr))
}
```